src/main/java/io/redlink/more/data/repository/StudyRepository.java

```diff
@@ -431,23 +427,17 @@ private static MapSqlParameterSource toParameterSource(long studyId, int partici
 
     private static MapSqlParameterSource toParameterSource(long studyId, int participantId, ParticipantConsent.ObservationConsent consent) {
         return toParameterSource(studyId, participantId)
-                .addValue("observation_id", consent.observationId())
-                ;
+                .addValue("observation_id", consent.observationId());
     }
 
 
-    public List<Interval> getIntervals(Long studyId, Integer participantId, RelativeEvent event) {
-        try (var stream = jdbcTemplate.queryForStream(
-                GET_PARTICIPANT_INFO_AND_START_DURATION_END_FOR_STUDY_AND_PARTICIPANT,
-                (rs, rowNum) -> {
-                    Instant start = rs.getTimestamp("start").toInstant();
-                    return Interval.fromRanges(SchedulerUtils.parseToObservationSchedulesForRelativeEvent(event, start));
-                },
-                studyId, participantId
-        )) {
-            return stream
-                    .flatMap(List::stream)
-                    .collect(Collectors.toList());
+    public Instant getStudyStartFor(Long studyId, Integer participantId) {
+        try {
+            return jdbcTemplate.queryForObject(
+                    GET_PARTICIPANT_INFO_AND_START_DURATION_END_FOR_STUDY_AND_PARTICIPANT,
+                    (rs, rowNum) -> rs.getTimestamp("start").toInstant(),
+                    studyId, participantId
+            );
         } catch (Exception e) {
             throw new BadRequestException("Failed to retrieve intervals for studyId: " + studyId + " and participantId: " + participantId);
```

Comment:
I'd rather not throw an exception here but return null or wrap it in an optional